server/portal/apps/datafiles/handlers/googledrive_handlers.py

```diff
@@ -0,0 +1,29 @@
+from portal.libs.googledrive import operations
+from django.core.exceptions import PermissionDenied
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+allowed_actions = {
+    'private': ['listing', 'search', 'copy'],
+    'public': [],
+    'community': [],
```

Comment:
would another scheme end up here somehow?

Reply:
No, but if a call is somehow made with the wrong scheme it will return the correct error code.